pajbot/bot.py

```diff
@@ -646,6 +646,11 @@ def whisper(self, user, message):
             self.irc.whisper(user.login, message)
         if self.whisper_output_mode == WhisperOutputMode.CHAT:
             self.privmsg(f"{user}, {message}")
+        if self.whisper_output_mode == WhisperOutputMode.CONTROL:
+            chub = self.config["main"].get("control_hub", None)
+            if chub is not None:
+                chub = f"#{chub}"
+            self.privmsg(f"{user}, {message}", chub)
```

Comment:
What happens here if there is no `control_hub` configured in the config?

Reply:
privmsg is called with None as target channel, so it sends the message to the streamers channel by default. Could throw an error instead eShrug

Reply:
You should probably do nothing if no control hub is configured and print a warning on the console along the lines of ``Whisper output mode set to `control_hub` but no control hub configured in config, following whisper will not be sent: To {user}: {message}``